src/main/java/com/google/sps/data/User.java

```diff
@@ -5,12 +5,19 @@
 
 public final class User {
 
-  private final String name;
-  private final Collection<String> productArea;
+  private long id;
+  private String profileType;
+  private String name;
+  private String school;
+  private String major;
+  private String company;
+  private String occupation;
+  private String specialty;
```

Comment:
Why is there a String for specialty and a Collection for specialties, is there a difference?

Reply:
Specialty is coming from the option drop down in the form. 
Specialties would refer to the multiple specialties option

Reply:
Could you clarify what the "multiple specialties option" is?

Reply:
Specialties is a option that mentors choose multiple specialties they have